code/obj/machinery/clonepod.dm

```diff
@@ -679,6 +688,22 @@ TYPEINFO(/obj/machinery/clonepod)
 		src.light.disable()
 		src.UpdateIcon()
 
+	proc/remove_upgrade_modules(mob/user)
+		if (src.is_speedy)
+			src.is_speedy = FALSE
+			src.speed_bonus /= 3
+			src.meat_used_per_tick /= 4
+			var/obj/speedy = new /obj/item/cloneModule/speedyclone(src.loc)
+			src.visible_message(SPAN_ALERT("The [speedy] module falls to the floor!"))
+			playsound(src.loc, 'sound/effects/pop.ogg', 80, FALSE)
+		if (src.is_efficient)
+			src.is_efficient = FALSE
+			src.meat_used_per_tick /= 0.5
+			var/obj/efficient = new /obj/item/cloneModule/efficientclone(src.loc)
+			src.visible_message(SPAN_ALERT("The [efficient] module falls to the floor!"))
+			playsound(src.loc, 'sound/effects/pop.ogg', 80, FALSE)
```

Comment:
This would play twice if it had both. Not sure exactly what that does to the sound.